api/v1alpha1/pulp_types.go

```diff
@@ -490,19 +489,6 @@ type Web struct {
 	PDB *policy.PodDisruptionBudgetSpec `json:"pdb,omitempty"`
 }
 
-type ExternalDB struct {
-	PostgresPort     int    `json:"postgres_port,omitempty"`
-	PostgresSSLMode  string `json:"postgres_ssl_mode,omitempty"`
-	PostgresHost     string `json:"postgres_host,omitempty"`
-	PostgresUser     string `json:"postgres_user,omitempty"`
-	PostgresPassword string `json:"postgres_password,omitempty"`
-	PostgresDBName   string `json:"postgres_db_name,omitempty"`
-
-	// +kubebuilder:validation:Optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	PostgresConMaxAge string `json:"postgres_con_max_age,omitempty"`
-}
-
```

Comment:
why removing?

Reply:
I should have removed it in https://github.com/pulp/pulp-operator/pull/633 (instead of "exposing" the external db credentials/connection info through Pulp CR we are getting them from a secret).

My bad for doing several non-related modifications in this same commit :(